src/components/ComponentCard.jsx

```diff
@@ -3,12 +3,10 @@ import Link from 'next/link';
 export default function ComponentCard({ name, slug, image }) {
   return (
     <Link href={slug}>
-      <a className="component-link">
-        <div className="component-card">
-          <span className="component-card__header">{name}</span>
-          <img src={image} alt="" className="component-card__image" />
-        </div>
-      </a>
+      <div className="component-card component-link">
+        <span className="component-card__header">{name}</span>
+        <img src={image} alt="" className="component-card__image" />
+      </div>
```

Comment:
This might need to be refactored in some way. Maybe we can use an extra wrapper again (because we lost the `a` tag) for the `component-link` class. All of the styles are missing right now: 


<img width="1003" alt="Screen Shot 2022-11-01 at 10 44 36 AM" src="https://user-images.githubusercontent.com/69602589/199262303-e2a32762-cdbc-4cdc-b77a-eebb0fceb001.png">